language/llama3-405b/README.md

```diff
@@ -9,56 +9,6 @@
 
```

Comment:
@pgmpablo157321 can you change the run_offline.sh and run_accuracy.sh to correct the run command?